sys/linux/dev_kvm.txt

```diff
@@ -254,9 +254,17 @@ syzos_api_code {
 	ret	const[0xd65f03c0, int32]
 } [packed]
 
+syzos_api_msr {
+	call		const[2, int64]
+	size		bytesize[parent, int64]
+	arg_reg		flags[kvm_regs_arm64_sys, int64]
```

Comment:
Why do we pass only kvm_regs_arm64_sys here and not the other classes?

Reply:
Because this class is the one that contains system registers for which MSR/MRS instructions are allowed.